src/js/facility-locator/components/AppointmentInfo.jsx

```diff
@@ -26,7 +26,9 @@ export default class AppointmentInfo extends Component {
 
     const healthAccessAttrs = facility.attributes.access.health;
     const specialtyKeys = healthAccessAttrs && Object.keys(healthAccessAttrs);
-    delete specialtyKeys.primaryCare;
+    pull(specialtyKeys, 'primaryCare');
+    pull(specialtyKeys, 'effectiveDate');
```

Comment:
Thanks will make those changes in https://github.com/department-of-veterans-affairs/vets-website/pull/5749